web/src/hooks/useRouterSync.ts

```diff
@@ -6,8 +6,8 @@ const useRouterSync = () => {
   const params = useParams();
 
   useEffect(() => {
-    return RouterMiddleware.startListening(params);
-  }, [params]);
+    return RouterMiddleware.startListening({testId: params.testId, runId: params.runId});
+  }, [params.runId, params.testId]);
```

Comment:
We have to consider that params can mean multiple things in the future, not only `testId` and `runId`